### ISSUE TITLE
How to manage multiple reference.bib file

### ISSUE THREAD

**Issue by acasner** [PierreSenellart/erc-latex-template#2]
Hello,

I like your template and plan to use it for next Call.

Is there a way to use multiple references.bib files ( I am used to manage huge different .bib)
I have tried to add other bib ressources, but not able to make the new references within the third bib files appearing when compiling

\addbibresource{../self.bib}
\addbibresource{../references.bib}
\addbibresource{../Ablationfront_turbulence_2020.bib}

Thanks !

**Comment by PierreSenellart** [comment]
There is no reason why this should be an issue, the document class does not treat additionnal bibliography databases in any special way.

I have just tested it, this works fine. Make sure you use `biber` and not `bibtex`; the easiest is to use `latexmk -pdf` to compile the entire document, it will call `biber` as well as the LaTeX engine the right amount of times.

If you are still encountering issues, I will need to see an example of document that fails to compile.

**Comment by acasner** [comment]
Hello, 

Sorry, I don't get it: I have added 2 other bib files in the common, and it works under Overleaf 
But I don' know WHY: when I add 2 others, it fails AGAIN. 

Should I compile the common (I don't think so) , or solely B1 and B2 

It seems erratic to me (I didn't understand WHY the 2 first packages added were working !) 

https://www.overleaf.com/7859961775gnkjbxjrdyvc 

Here is the link to overleaf: 
%\addbibresource{Turbulence.bib} 
%\addbibresource{Ablationfront_turbulence_2020.bib} 


\addbibresource{self.bib} 
\addbibresource{references.bib} 
\addbibresource{Imaging.bib} 
\addbibresource{NIC_2020.bib} 
%\addbibresource{Turbulence.bib} 
%\addbibresource{Ablationfront_turbulence_2020.bib} 




De: "Pierre Senellart" <notifications@github.com> 
À: "PierreSenellart/erc-latex-template" <erc-latex-template@noreply.github.com> 
Cc: "Alexis Casner" <alexis.casner@u-bordeaux.fr>, "Author" <author@noreply.github.com> 
Envoyé: Dimanche 7 Juin 2020 16:25:37 
Objet: Re: [PierreSenellart/erc-latex-template] How to manage multiple reference.bib file (#2) 





There is no reason why this should be an issue, the document class does not treat additionnal bibliography databases in any special way. 

I have just tested it, this works fine. Make sure you use biber and not bibtex ; the easiest is to use latexmk -pdf to compile the entire document, it will call biber as well as the LaTeX engine the right amount of times. 

If you are still encountering issues, I will need to see an example of document that fails to compile.

**Comment by acasner** [comment]
OK, thanks a lot. 

Disregard my email. 




De: "Alexis Casner" <alexis.casner@u-bordeaux.fr> 
À: "PierreSenellart" <reply@reply.github.com> 
Envoyé: Lundi 8 Juin 2020 12:21:50 
Objet: Re: [PierreSenellart/erc-latex-template] How to manage multiple reference.bib file (#2) 

Hello, 

Sorry, I don't get it: I have added 2 other bib files in the common, and it works under Overleaf 
But I don' know WHY: when I add 2 others, it fails AGAIN. 

Should I compile the common (I don't think so) , or solely B1 and B2 

It seems erratic to me (I didn't understand WHY the 2 first packages added were working !) 

https://www.overleaf.com/7859961775gnkjbxjrdyvc 

Here is the link to overleaf: 
%\addbibresource{Turbulence.bib} 
%\addbibresource{Ablationfront_turbulence_2020.bib} 


\addbibresource{self.bib} 
\addbibresource{references.bib} 
\addbibresource{Imaging.bib} 
\addbibresource{NIC_2020.bib} 
%\addbibresource{Turbulence.bib} 
%\addbibresource{Ablationfront_turbulence_2020.bib} 




De: "Pierre Senellart" <notifications@github.com> 
À: "PierreSenellart/erc-latex-template" <erc-latex-template@noreply.github.com> 
Cc: "Alexis Casner" <alexis.casner@u-bordeaux.fr>, "Author" <author@noreply.github.com> 
Envoyé: Dimanche 7 Juin 2020 16:25:37 
Objet: Re: [PierreSenellart/erc-latex-template] How to manage multiple reference.bib file (#2) 





There is no reason why this should be an issue, the document class does not treat additionnal bibliography databases in any special way. 

I have just tested it, this works fine. Make sure you use biber and not bibtex ; the easiest is to use latexmk -pdf to compile the entire document, it will call biber as well as the LaTeX engine the right amount of times. 

If you are still encountering issues, I will need to see an example of document that fails to compile.
